packages/@okta/vuepress-site/docs/guides/oie-embedded-sdk-use-cases/oie-embedded-sdk-use-case-pwd-recovery-mfa/nodejs/integrationsteps.md

```diff
@@ -1 +1,95 @@
+
+After the user clicks **Submit** to start the reset flow, the next step is to call `idx.recoverPassword` and pass in the email address captured from the form. This starts the password recovery flow and returns a status of `Idx.Status:PENDING` and a `nextStep` field with inputs for an authenticator type.
+
+The call is shown in `recover-password.js` file of the SDK sample application.
+
+```JavaScript
+router.post('/recover-password', async (req, res, next) => {
+  const { username } = req.body;
+  const authClient = getAuthClient(req);
+  const transaction = await authClient.idx.recoverPassword({ username });
+  handleTransaction({ req, res, next, authClient, transaction });
+});
+```
+
+The `handleTransaction` response, in the `handleTransaction.js` page, directs the user to a select authenticators page (`/select-authenticator` in this example) that the user needs to use to select an authentication type. In this case, the email factor is configured.
+
+```JavaScript
+// authenticator authenticate
+    case 'select-authenticator-authenticate':
+      redirect({
+        req, res, path: '/select-authenticator'
+      });
+      return true;
+```
+
+>**NOTE:** Review the full use of `idx.recoverPassword` in the [Okta Auth JS SDK](https://github.com/okta/okta-auth-js/blob/master/docs/idx.md#idxrecoverpassword). Other uses of `idx.recoverPassword` are documented in the following steps.
+
+### Step 2: Create reset password authenticators page
+
+The next step is to create a page that shows the authenticator that is returned from the `handleTransaction` method. For this use case, it shows the email authenticator. The page includes the name of the authenticator and the ability to select the authenticator to initiate the authentication verification process.
+
+After the user selects the email authenticator, the next step is to call `idx.recoverPassword` with the authenticator type, as shown in the `authenticator.js` file of the SDK sample application. The method returns a status of `Idx.Status:PENDING`. This status indicates that the Okta platform has emailed the verification code to the user's email address and it's now awaiting verification. The response also includes a `nextStep` field that requires an email verification code input parameter.
+
+```JavaScript
+router.post('/select-authenticator', async (req, res, next) => {
+  const { idxMethod } = req.getFlowStates();
+  const { authenticator } = req.body;
+  const authClient = getAuthClient(req);
+  const transaction = await authClient.idx[idxMethod]({ authenticator });
+  handleTransaction({ req, res, next, authClient, transaction });
+});
+```
+
+### Step 4: Create code verification page
+
+After the call to `idx.recoverPassword` has responded with a status of `Idx.Status:PENDING`, the user is redirected to a code verification page. The page must display a field to enter a code and a button or link to send the code to Okta for the email verification.
```

Comment:
```suggestion
After the call to `idx.recoverPassword()` has responded with a status of `Idx.Status:PENDING`, redirect the user to a code verification page. This page must display a field to enter a code and a button or link to send the code to Okta for the email verification.
```

Reply:
`methodname()` format is normally used for calling out methods with parameters. But if this is not standard convention for node.js, please ignore this suggestion.

Reply:
I'm confirming. I've been simply stating to call it, and that's how I've done it all through my use-cases. I'll leave for now, but will clean this up on second pass.